include/deal.II/numerics/data_out.h

```diff
@@ -274,14 +274,14 @@ class DataOut : public DataOut_DoFData<DH, DH::dimension, DH::space_dimension>
    * pair that is locally owned. If this object operates on a non-distributed
    * triangulation, the result equals what first_cell() returns.
    */
-  cell_iterator first_locally_owned_cell ();
+  virtual cell_iterator first_locally_owned_cell ();
```

Comment:
what's the reason this and the next function is made `virtual`? it simply wraps around `first/next_cell` with a fairly obvious implementation. I don't see why anyone would want to override it.